recipe/meta.yaml

```diff
@@ -14,6 +14,7 @@ requirements:
   build:
     # this dep is for us to be able to use gcc -dumpmachine to get the CBUILD value
     - gcc_impl_{{ target_platform }}
+    - shellcheck
```

Comment:
Are there any downsides in adding additional build-time dependencies here?

Reply:
Not that I know of.